### PR TITLE
Unification of error messages for e2e test

### PR DESCRIPTION
**Pull request by akoniar**

### CODE REVIEW

**Comment by akoniar** [comment]
@lholmquist Please, code review

File: app.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const db = require('./lib/db');
 
 const fruits = require('./lib/routes/fruits');
 
-app.use(bodyParser.json());
+app.use(bodyParser.text({ type: 'application/json' }));
```
Comment (lholmquist) [inline app.js]: why do you need to do this.  isn't this just another way of writing `bodyParser.json()`?

File: lib/routes/fruits.js [hunk 2]
```diff
@@ -38,23 +38,32 @@ router.post('/fruits', validations.validateCreate, (request, response) => {
...
-  fruits.update({name, stock, id}).then((result) => {
+router.put('/fruits/:id', [parseRequestBodyToJson, validations.validateCreateUpdateRequest], (request, response) => {
+  const { name, stock } = request.body;
+  const { id } = request.params;
+  fruits.update({ name, stock, id }).then((result) => {
     if (result.rowCount === 0) {
       response.status(404);
       return response.send(`Unknown item ${id}`);
     }
-    return response.sendStatus(204);
```
Comment (lholmquist) [inline lib/routes/fruits.js]: i don't think there is any reason to remove that `return` .  it technically doesn't need to be there, but the other methods have it, so we should keep it for consistency

File: app.js (hunk 1 above)
Comment (akoniar) [reply app.js]: Good point. I look at on it.

File: lib/routes/fruits.js (hunk 2 above)
Comment (akoniar) [reply lib/routes/fruits.js]: Ok.

**Comment by lholmquist** [comment]
@akoniar looks like travis is broken

**Comment by lholmquist** [comment]
@akoniar still broken.  you should probably run `npm run test` before pushing up

**Comment by lholmquist** [comment]
@akoniar landed as 50a7f72

**Pull request closed without merging**